eng/.docsettings.yml

```diff
@@ -7,6 +7,7 @@ omitted_paths:
   - sdk/keyvault/azure-keyvault/*
   - doc/*
   - sdk/**/samples/*
+  - "sdk/identity/azure-identity/tests/*"
```

Comment:
Why you need  '"' while others don't?

Reply:
Yeah, you don't need the quotes.

Reply:
Kubernetes got me into the habit of quoting values in YAML. I didn't even notice 😊

I'll change this one to be consistent.